lib/Runtime/Language/JavascriptStackWalker.cpp

```diff
@@ -617,14 +617,14 @@ namespace Js
         return nullptr;
     }
 #if ENABLE_NATIVE_CODEGEN
-    void JavascriptStackWalker::WalkAndClearInlineeFrameCallInfoOnException()
+    void JavascriptStackWalker::WalkAndClearInlineeFrameCallInfoOnException(void *tryCatchFrameAddr)
     {
         // Walk the stack and when we find the first JavascriptFrame, we clear the inlinee's callinfo for this frame
         // It is sufficient we stop at the first Javascript frame which had the enclosing try-catch
         // TODO : Revisit when we start inlining functions with try-catch/try-finally
         while (this->Walk(true))
         {
-            if (this->IsJavascriptFrame())
+            if (JavascriptFunction::IsNativeAddress(this->scriptContext, this->currentFrame.GetInstructionPointer()))
             {
                 if (!this->isNativeLibraryFrame)
```

Comment:
>isNativeLibraryFrame [](start = 27, length = 20)

is this check valid for our builtins implementation in javascript? What if they have a try catch and try throws?

Reply:
isNativeLibraryFrame is true for C++ library frames only. The check is actually a no-op now, since such frames fail the IsNativeAddress check. Thanks for catching this, I'll remove the redundant condition.